src/main/java/com/mathworks/ci/RunMatlabTestsStep.java

```diff
@@ -142,6 +156,18 @@ private String getInputArgs() {
         args.put("SimulinkTestResults", getTestResultsSimulinkTest());
         args.put("CoberturaCodeCoverage", getCodeCoverageCobertura());
         args.put("CoberturaModelCoverage", getModelCoverageCobertura());
+        if(!getSourceFolder().isEmpty()){
+            args.put("SourceFolder", getCellArrayFrmList(getSourceFolder()));
+        }
         return args;
     }
+
+    private String getCellArrayFrmList(List<String> listOfStr){
+        // Ignore empty string values in the list
+        Predicate<String> isEmpty = String::isEmpty;
+        Predicate<String> isNotEmpty = isEmpty.negate();
+        List<String> filteredListOfStr = listOfStr.stream().filter(isNotEmpty).collect(Collectors.toList());
+        filteredListOfStr.replaceAll(val -> "'" + val.replaceAll("'", "''") + "'");
+        return "{" + String.join(",", filteredListOfStr) + "}";
+    }
```

Comment:
Can you avoid repeating this function?

Reply:
would be good idea to just start a Util class it would be a good start atleast with one method in it. looks like getting cell array from list may be resused for testfoldrselction later.

Reply:
I had something similar in an earlier commit https://github.com/mathworks/jenkins-matlab-plugin/pull/148/commits/d1c2fca54e6000c51007edcd95b7c18df8cfb6a3 . I'll create a new class 'Utilities' and add this method there.